ord_schema/proto/reaction.proto

```diff
@@ -822,28 +815,26 @@ message ReactionOutcome {
   Percentage conversion = 2;
```

Comment:
I know this refers to a starting material and not a product, but it almost feels like this is well-suited as a `ProductMeasurement.MeasurementType` since it is useful to know how the conversion was measured (NMR, LC, etc.). If the limiting reactant is being quantified (which it has to be to report conversion), it could technically be included in `ReactionOutcome.products`, in which case the `ProductMeasurement.MeasurementType = CONVERSION` could be specified along with the `analysis_key` (I think we haven't really decided on how to handle measurements of remaining starting materials).

Reply:
I agree that it's useful to know how it was reported. There's also the complication that one can quantify the remaining amounts of multiple starting materials, which a single `conversion` field doesn't address

Reply:
Added a comment about including multiple starting materials as "products" if individually quantified